README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+My name is Gabrielle Meyers, but most people call me Gabby.
+
+## Quick Facts
+* I'm a rising third year at the College.
+* I'm majoring in Public Policy and History.
+* I'm from Long Island, NY, where I currently live with my family and my dog, Ella.
```

Comment:
Nice use of itemized list! I recommend leaving a blank line between different Markdown elements, like this:

```
## Quick Facts

* I'm a rising third year at the College.
* I'm majoring in Public Policy and History.
* I'm from Long Island, NY, where I currently live with my family and my dog, Ella.
```

It avoids rendering issues in more complex documents